pup-0006.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Commit Bit Assignment
+author: Ina Panova
+created: 15-08-2018
+status: Approved
+version: 1
+---
+
+## Summary
+
+For pulpcore and its related tools, there is no written process to describe giving the commit bit
```

Comment:
I like the comment for clarification that "area" means repository on line 62.  I suggest we start general (s/pulpcore and its related tools/the Pulp project/) - I'll give specific comments to be more specific below.

Reply:
yeah I like the "Pulp project repositories" wording more